src/requirements/courses-json-generator.ts

```diff
@@ -32,6 +32,14 @@ const getSubjects = async (semester: string): Promise<readonly string[]> => {
   }
 };
 
+const cleanString = (value: string | null) => {
+  if (value === null) return '';
+  return value.replace(/\u00a0/g, ' ');
```

Comment:
Can you refactor `/\u00a0/g` as a constant?

Reply:
sure